demo-notebooks/additional-demos/local_interactive.ipynb

```diff
@@ -49,12 +49,10 @@
    "outputs": [],
    "source": [
     "# Create our cluster and submit appwrapper\n",
-    "namespace = \"default\"\n",
+    "namespace = \"default\" # Update to your namespace\n",
```

Comment:
Should we be consistent in how we provide the namespace in each notebook? Sometimes we define it before cluster config and other times we don't.

Reply:
For demos where we define it before config we use the `namespace` & `cluster_name` vars for generating certs later.